src/common/set.c

```diff
@@ -347,6 +347,24 @@ util_replica_force_page_allocation(struct pool_replica *rep)
 	}
 }
 
+/*
+ * util_map_part_type -- memory map part file into memory according
+ * to the type of file.
+ */
+static void *
+util_map_part_type(struct pool_set_part *part, void *addr, size_t len,
+	int prot, int flags, size_t offset, int *map_sync)
+{
+	if (part->is_dev_dax) {
+		*map_sync = 0;
+		return mmap(addr, len, prot, flags, part->fd,
+				(os_off_t)offset);
+	}
+
+	return util_map_sync(addr, len, prot, flags, part->fd,
```

Comment:
what happens when you call mmap /w MAP_SYNC on devdax?

`return util_map_sync(addr, len, prot, flags, part->fd, (os_off_t)offset, part->is_dev_dax ? NULL : map_sync);`